### PR TITLE
Add new variable to cache file path + Possible bug fixed

### PR DESCRIPTION
**Pull request by ivandrofly**
- [ ] More test

### CODE REVIEW

File: src/Logic/Dictionaries/NamesList.cs [hunk 1]
```diff
@@ -159,13 +162,13 @@ public bool Remove(string name)
                     _namesMultiList.Remove(name);
 
                 var userList = new HashSet<string>();
-                var fileName = GetLocalNamesUserFileName();
-                LoadNamesList(fileName, userList, userList);
```
Comment (ivandrofly) [inline src/Logic/Dictionaries/NamesList.cs]: Shouldn't third parameter be `_namesMultiList`? /cc @niksedk

Comment (ivandrofly) [inline src/Logic/Dictionaries/NamesList.cs]: Shouldn't third parameter be `_namesMultiList`? /cc @niksedk

File: src/Logic/Dictionaries/NamesList.cs [hunk 2]
```diff
@@ -159,13 +162,13 @@ public bool Remove(string name)
...
+                _userFile = _userFile ?? GetLocalNamesUserFileName(); // Try get file name if _userFile if null
```
Comment (ivandrofly) [inline src/Logic/Dictionaries/NamesList.cs]: No loop needed for every add/remove name :wink:

File: src/Logic/Dictionaries/NamesList.cs (hunk 1 above)
Comment (niksedk) [inline src/Logic/Dictionaries/NamesList.cs]: Normally yes... but in "Remove" it's intended to load all entries in just one list

Comment (ivandrofly) [inline src/Logic/Dictionaries/NamesList.cs]: :+1: Will do quick update then

Comment (ivandrofly) [inline src/Logic/Dictionaries/NamesList.cs]: Reverted /cc @niksedk

**Pull request closed without merging**